torch/nn/functional.py

```diff
@@ -339,16 +339,12 @@ def fractional_max_pool2d_with_indices(input, kernel_size, output_size=None,
                          "an output_size or an output_ratio")
     if output_size is None:
         _output_ratio = _pair(torch.jit._unwrap_optional(output_ratio))
```

Comment:
missing one

Reply:
This function throws an error if both output_size is None and output_ratio is None. 

Then it checks if output_size is None, which therefore means output_ratio is not none. Since the NoneType inference doesn't do any reasoning like "either output_ratio or output_none is None" I included the manual unwrap. I could remove it by instead writing:

`
if output_size is None and output_ratio is not None:
`